internal/core/domain/claim.go

```diff
@@ -36,10 +36,10 @@ type Claim struct {
 	CoreClaim        CoreClaim       `json:"core_claim"`
 	MTPProof         pgtype.JSONB    `json:"mtp_proof"`
 	SignatureProof   pgtype.JSONB    `json:"signature_proof"`
-	IdentityState    *string         `json:"-"`
+	IdentityState    *string         `json:"identity_state"`
```

Comment:
Probably we do not need to export it as json now, as we are only adding the ID of the claim and not all the data.

Reply:
agree, I added them as I was sending the credential in the pubsub before

Reply:
agree, I added them as I was sending the credential in the pubsub before